modules/virt-importing-vm.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
+// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
+[id='virt-importing-vm_{context}']
+ifdef::virt-importing-vmware-vm[]
+= Importing a VMware virtual machine or template
+
+You can import a VMware virtual machine or template using the virtual machine wizard.
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+= Importing a Red Hat Virtualization virtual machine
+
+You can import a Red Hat Virtualization (RHV) virtual machine using the virtual machine wizard.
+endif::[]
+
+.Prerequisites
+
+ifdef::virt-importing-vmware-vm[]
+* You must create a VDDK image, push it to an image registry, and add it to the `v2v-vmware` ConfigMap.
+* The VMware virtual machine must be powered off.
+endif::[]
+
+* There must be sufficient storage space for the imported disk.
++
+[WARNING]
+====
+If you try to import a virtual machine whose disk size is larger than the available storage space, the operation cannot complete. You will not be able to import another virtual machine or to clean up the storage because there are insufficient resources to support object deletion. To resolve this situation, you must add more object storage devices to the storage backend.
+====
+
+.Procedure
+
+. In the web console, click *Workloads* -> *Virtual Machines*.
+. Click *Create Virtual Machine* and select *Import with Wizard*.
+
+ifdef::virt-importing-vmware-vm[]
+. In the *Connect to Provider* screen, select *VMware* from the *Provider* list.
+. Select *Connect to New Instance* or a saved vCenter instance from the *vCenter instance* list.
+
+* If you select *Connect to New Instance*, enter the *vCenter hostname*, *Username*, and *Password*.
+* If you select a saved vCenter instance, the wizard connects to the vCenter instance using the saved credentials.
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+. In the *Connect to Provider* screen, select *Red Hat Virtualization (RHV)* from the *Provider* list.
+. Select *Connect to New Instance* or a saved RHV instance from the *RHV Instance* list.
+
+* If you select *Connect to New Instance*, fill in the following fields:
+
+** *API URL*: For example, `\https://<RHV_Manager_FQDN>:8443/ovirt-engine/api`
```

Comment:
The \ before http is not needed.

Reply:
It is required. Examples of URLs need to have an escape character (\) so that they do not show up as bad URLs when a link check is run. The `\` does not show up in the generated build.